src/Login.js

```diff
@@ -35,7 +37,15 @@ export class Login extends React.Component {
                     </span>
                     <input name="password" value={this.state.password} onChange={this.onChange}/>
                 </div>
-                <button onClick={this.onClick}>Click me</button>
+                <button onClick={this.onClick}>Log in</button>
+                <Route render={({ history}) => (
```

Comment:
When you render the component like this `<Route path="/login" component={Login}/>` you already have `history` in the props. You can simplify it to:
```js
<button onClick={() => this.props.history.push('/regisster')}>Sign up</button>
```